packages/mobile/src/alert/reducer.ts

```diff
@@ -16,10 +17,21 @@ const initialState = null
 export const reducer = (state: State | null = initialState, action: ActionTypes): State | null => {
   switch (action.type) {
     case Actions.SHOW:
-      const { type, alertType, ...other } = action
       return {
-        type: alertType,
-        ...other,
+        errorType: 'show',
+        type: action.alertType,
+        message: action.message,
+        dismissAfter: action.dismissAfter,
+        buttonMessage: action.buttonMessage,
+        title: action.title,
+        underlyingError: action.underlyingError,
+      }
+    case Actions.LOG:
```

Comment:
Good catch. It can be consolidated into one with the `errorType` parameter used to differentiate.

Reply:
Settled on consolidating on one action but still having separate dispatching functions called `showError` and `showErrorInline`. Changing `showError` is in undertaking in and of itself because it's so widely used and already stacked with optional parameters. Felt cleaner to make a separate function rather than pile onto `showError`.